platform/core/src/extensions/ExtensionManager.ts

```diff
@@ -340,6 +340,7 @@ export default class ExtensionManager {
 
       return extensionModule;
     } catch (ex) {
+      console.log('Exception:', ex);
```

Comment:
Need more context, something along `Extension Manager::get:: ....` . So why we do need this since the error below has enogh information, maybe just add the exception there

Reply:
Actually removed context - this is a developer level thing just to ensure the stack trace is preserved when thigns go wrong, so the exception message alone is sufficient.